Longest_palindrome.py

```diff
@@ -0,0 +1,30 @@
+#// Time Complexity : O(n)
+#// Space Complexity : O(n)
```

Comment:
review space again

Reply:
Hi Nikhil,

I am sorry about it .It is O(1) as the dictionary contains at max only 52 characters which is a constant.